### PR TITLE
feat: support v2 volume live migration

### PR DESCRIPTION
**Pull request by PhanLe1010**
longhorn/longhorn#6361

### CODE REVIEW

**Review by james-munson — APPROVED** [review]
LGTM.  One question.

**Comment by derekbit** [comment]
Hello @PhanLe1010 
Conventional Commits / commit-lint (pull_request_target)  check failed.

**Review by derekbit — DISMISSED** [review]
LGTM. The implementation is quite straightforward. Thank you.

**Comment by derekbit** [comment]
Hello @PhanLe1010 
We can merge the PR after addressing the comments. Thank you.

**Comment by PhanLe1010** [comment]
> Hello @PhanLe1010 We can merge the PR after addressing the comments. Thank you.

Thank you. I am fixing it

File: k8s/crds.yaml [hunk 1]
```diff
@@ -19,8 +19,7 @@ spec:
   scope: Namespaced
   versions:
   - additionalPrinterColumns:
-    - description: The current state of the pod used to provision the backing image
-        file from source
+    - description: The current state of the pod used to provision the backing image file from source
```
[the remark this answers is not in the document]
Comment (PhanLe1010) [reply k8s/crds.yaml:22]: I am not sure which one change it. Maybe newer version of controller-gen change it?

**Comment by PhanLe1010** [comment]
All fixed. Thanks @derekbit

**Review by derekbit — APPROVED** [review]
LGTM

**Review by derekbit — APPROVED** [review]

**Pull request merged**